### DISCUSSION TITLE
SSD1327 96x96 SPI support

### DISCUSSION THREAD

**Discussion by JamesYan99** [olikraus/u8g2#2439] [General]
Hi Oliver,
Thanks for your brilliant U8g2lib. 
I am using Zhongjingyuan SSD1327 based 96x96 OLED with SPI interface for my project and noticed that currently supported device list shows only SSD1327 128x128 for Waveshare OLED. I use this constructor for communicating the display but the characters are shifted and some characters cannot be displayed. 

Could you also consider supporting the 96x96 resolution as well?

Warm regards,
James

**Comment by JamesYan99** [comment]
Hi Oliver,
  
   I uses the following constructor for seeed_96x96 display, and the example runs correctly. Thanks for your time and the great library.
   U8G2_SSD1327_SEEED_96X96_F_4W_HW_SPI u8g2(U8G2_R0, /* cs=*/ 5, /* dc=*/ 17, /* reset=*/ 16);

Thanks and best regards,

Jim.

**Reply by olikraus** [reply]
Thanks for the feedback. Great you found a fitting constructor.
